src/index/hnsw.cpp

```diff
@@ -129,6 +129,7 @@ HNSW::build(const DatasetPtr& base) {
         std::vector<int64_t> failed_ids;
         {
             SlowTaskTimer t("hnsw graph");
+            //#pragma omp parallel for
```

Comment:
add TODO ?

Reply:
delete it, we use threadpool for parallel